testing/liquibase/APKBUILD

```diff
@@ -1,19 +1,24 @@
 # Contributor: Curt Tilmes <Curt.Tilmes@nasa.gov>
 # Maintainer: Curt Tilmes <Curt.Tilmes@nasa.gov>
 pkgname=liquibase
-pkgver=3.5.3
+pkgver=3.6.1
 pkgrel=0
 pkgdesc="Source Control for your Database"
 url="http://www.liquibase.org/"
-arch="all"
+arch="noarch"
```

Comment:
Is this really "noarch"? https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#arch
Because looks it download binary files

Reply:
I haven't tested on other arches, but my understanding is that it is java byte-code that is executed by the VM on any architecture the same way.